src/docs/doc.yaml

```diff
@@ -2,7 +2,7 @@ paths:
   /api/auth/register:
     post:
       tags:
-        - register
+        - Local auth
```

Comment:
"Local authentication" may be fancier and cleaner :D

Reply:
new comment : in his last PR, Amjad change that part as "Authentication" for all the authentication process including local and 3rd party, it makes more sense to me so you can leave it like this and he can fix that in his merge or you can change this as "Authentication"